src/eventHandlers/AccountMetadataEmittedEvent/handlers/handleEcosystemMainAccountMetadata.ts

```diff
@@ -148,6 +148,14 @@ async function upsertEcosystemMainAccount({
         ? metadata.isVisible
         : true,
     lastProcessedVersion: makeVersion(blockNumber, logIndex).toString(),
+    color:
+      'color' in metadata
+        ? metadata.color
+        : unreachableError('Missing color in Ecosystem Main Account metadata'),
+    avatar:
```

Comment:
I think I called this out before, but maybe it didn't matter, but is it worth addressing the type property here?